### DISCUSSION TITLE
Tracking tamed animals

### DISCUSSION THREAD

**Discussion by Mortdredd** [coldino/Larkator#68] [Q&A]
Im hoping to use this to track some missing animals,  I can see they are still alive using ingame commands but cannot work out how to teleport me to them or them to me.  

the post on the forums says this can be used to track tamed animals but I cant seem to work out how.

**Comment by coldino** [comment]
In the bottom left of the window there's a button named something like "Show tames". That should do it for you.

You can also enable cheat commands in the settings window, to get that teleport command once you see the list of tames.
